### DISCUSSION TITLE
跨端扩展适配

### DISCUSSION THREAD

**Discussion by Ymm0008** [jdf2e/nutui#2011] [Ideas]
NutUI 已完成 H5 和 小程序组件建设，规划的 `uni-app`、`react-native`、`flutter` 技术栈 期待您的参与~

### uni-nutui

由社区开发者开发并维护的 [uni-app 版 NutUI](https://ext.dcloud.net.cn/plugin?id=9345) 已上线并开源，可通过 `uni-app` 插件市场搜索。欢迎您积极 PR 到 uni-nutui。

![uni-nutui](https://img13.360buyimg.com/imagetools/jfs/t1/196038/9/31999/64284/63b68de1F79e3be32/c28fa53d7257fc65.png)

### RN 版本

`RN` 版本的 NutUI 目前在京东内部开发中。

### 诚邀共建

`flutter` 技术的跨端适配，如您有更好的想法，可提出开发方案、组件交互设计等建设方案，为跨端组件库开发工作做准备。

对于 `uni-nutui`、`RN nutui` 的开发工作，感兴趣的开发者同学都可参加，开源项目需要大家的支持和贡献。

![vx](https://img13.360buyimg.com/imagetools/s200x200_jfs/t1/187308/9/29678/349561/634fa983E93012590/a641089fdfed5911.png)

**Comment by yang1206** [comment]
你好，uni-nutui的移植我觉得有些粗糙，有不少组件又不支持，大部分组件都有问题，而且还是基于 nutui3 的，所以我基于taro nutui4.x版本重新适配了 uniapp，支持所有组件，这是项目地址https://github.com/yang1206/uniapp-nutui
